index.html

```diff
@@ -2,295 +2,317 @@
-                <h3 class="skill-title">Languages</h3>
-                <button class="btn-chev" type="button">
-                  <img src="image/chev-dow.png" alt="" />
-                </button>
-              </div>
-              <div class="separator-line"></div>
-              <ul class="language-list">
-                <li>
-                  <img src="image/javascript.png" alt="JavaScript" />JavaScript</li>
-                <li>
-                  <img src="image/html.png" alt="HTML" />HTML</li>
-                <li>
-                  <img src="image/css.png" alt="CSS" />CSS</li>
-              </ul>
-            <!-- /languages -->
-            <li class="my-skill">
-              <div class="skill-container">
-                <h3 class="skill-title">Frameworks</h3>
-                <button class="btn-chev" type="button">
-                  <img src="image/chev-right.png" alt="" />
-                </button>
-              </div>
-              <div class="separator-line"></div>
+        </div>
+        <div class="about-skills">
+          <div class="skills about-skill">
+            <div class="icon">
+              <img src="images/Icons(1).png" alt="" />
+            </div>
+            <h4>Skills</h4>
+            <ul>
+              <li>CodeKit</li>
+              <li>GitHub</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+        </div>
+      </div>
+    </section>
+    <!-- Contact Section -->
+    <section class="contact-section">
+      <div class="container-contact">
+        <div class="container-contact2">
+          <h2 class="contact-description">
+            I am interested in knowing about new projects, so if you would like to communicate please get in touch with me.
+        </h2>
+        <form action="https://formspree.io/f/xayknkna" method="POST">
+          <div class="name-form">
+            <input
+            type="text"
+            name="firstName"
+            id="firstName"
+            placeholder="First Name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="text"
+            name="lastName"
+            id="lastName"
+            placeholder="Last Name"
+            maxlength="30"
+            required
+          />
+          </div>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+
+          <textarea
+            name="comment"
+            placeholder="Enter text here"
+            id="comment"
+            cols="30"
+            rows="10"
+            maxlength="500"
+          ></textarea>
+
+          <!-- your other form fields go here -->
```

Comment:
- [ ] kindly remove any unnecessary comments to make your code looks clean

Reply:
Thank you. That has been removed.